ext/reflection/php_reflection.stub.php

```diff
@@ -50,6 +50,9 @@ public function getClosureThis() {}
     /** @return ReflectionClass|null */
     public function getClosureScopeClass() {}
 
+    /** @return array */
+    public function getClosureUsedVariables() {}
```

Comment:
As this is a new method, it should get a proper `): array` type

Reply:
ah I missed that ... done ...